docs/product/shortcuts.md

```diff
@@ -5,70 +5,100 @@ category: product
-| <kbd>option</kbd> + <kbd>shift</kbd> + <kbd>LMB</kbd> click node name | Inverse node selection. |
-| <kbd>option</kbd> + <kbd>shift</kbd> + <kbd>LMB</kbd> drag background | Inverse nodes selection. |
-
-
+| <kbd>LMB</kbd> click node name                                                       | Deselect all nodes. Select the target node. |
+| <kbd>LMB</kbd> click background                                                      | Deselect all nodes. |
+| <kbd>LMB</kbd> drag background                                                       | Select nodes using selection-box (not working now). |
+| <kbd>shift</kbd> + <kbd>LMB</kbd> click node name                                    | Add / remove node to the selection group. |
+| <kbd>shift</kbd> + <kbd>LMB</kbd> drag background                                    | Add / remove nodes to the selection group (not working now. |
+| :warning: <kbd>*-arrow</kbd>                                                         | Select node on the right side of the newest selected node. |
+| :warning: <kbd>cmd</kbd> + <kbd>a</kbd>                                              | Select all nodes. |
+| :warning: <kbd>escape</kbd>                                                          | Deselect all nodes (if not in a mode, like edit mode). |
+| <kbd>shift</kbd> + <kbd>ctrl</kbd> + <kbd>LMB</kbd> click node name                  | Add node to the selection group. |
+| <kbd>shift</kbd> + <kbd>ctrl</kbd> + <kbd>LMB</kbd> drag background                  | Add nodes to the selection group (not working now. |
+| <kbd>shift</kbd> + <kbd>alt</kbd> + <kbd>LMB</kbd> click node name                   | Remove node to the selection group. |
+| <kbd>shift</kbd> + <kbd>alt</kbd> + <kbd>LMB</kbd> drag background                   | Remove nodes to the selection group (not working now. |
+| <kbd>shift</kbd> + <kbd>ctrl</kbd> + <kbd>alt</kbd> + <kbd>LMB</kbd> click node name | Inverse node selection. |
+| <kbd>shift</kbd> + <kbd>ctrl</kbd> + <kbd>alt</kbd> + <kbd>LMB</kbd> drag background | Inverse nodes selection (not working now. |
 
-### Node dragging
 
+#### Node Editing
 | Shortcut | Action |
 | -------- | ------ |
-| left-mouse-button drag non-selected node name | Move the node to new position. |
-| left-mouse-button drag selected node name     | Move all selected nodes the node to new positions. |
-
+| <kbd>shift</kbd>+<kbd>tab</kbd>                  | Show / hide node searcher. |
+| <kbd>backspace</kbd>                             | Remove selected nodes. |
+| <kbd>cmd</kbd>+<kbd>g</kbd>                      | Collapse (group) selected nodes. |
+| <kbd>meta</kbd>+<kbd>LMB</kbd>                   | Start editing node expression. |
+| <kbd>meta</kbd>+<kbd>enter</kbd>                 | Start editing node expression. |
+| <kbd>enter</kbd> or <kbd>LMB</kbd> on suggestion | Pick selected suggestion and commit editing. |
```

Comment:
Im dropping it because to me, when you press arrow up and you do not "select suggestion from the list", you are selecting the "current suggestion" , which for me still holds here.

I'm adding tab!

Reply:
No, shift+tab brings searcher, but when you have suggestion selected Tab should add it to the input and continue editing.

And no, I don't agree "you are selecting the "current suggestion" " - the searcher input is not suggestion, because it may be not suggested in any way to user, but solely his input. The "suggestion" term is solely used for suggestions on the list.